docs/install.md

```diff
@@ -416,7 +416,8 @@ data:
 
 Alpha features are still in development and their syntax is subject to change.
 To enable these, set the `enable-api-fields` feature flag to `"alpha"` in
-the `feature-flags` ConfigMap alongside your Tekton Pipelines deployment.
+the `feature-flags` ConfigMap alongside your Tekton Pipelines deployment via
+`kubectl edit configmap feature-flags -n tekton-pipelines`.
```

Comment:
NIT: maybe we could even say:

```shell
kubectl patch cm feature-flags -n tekton-pipelines -p '{"data":{"enable-api-fields":"alpha"}}'
```

Reply:
Done: https://github.com/tektoncd/pipeline/pull/5437